op-e2e/actions/garbage_channel_out.go

```diff
@@ -253,12 +254,16 @@ func blockToBatch(block *types.Block) (*derive.BatchData, error) {
 	}
 
 	return &derive.BatchData{
-		BatchV1: derive.BatchV1{
-			ParentHash:   block.ParentHash(),
-			EpochNum:     rollup.Epoch(l1Info.Number),
-			EpochHash:    l1Info.BlockHash,
-			Timestamp:    block.Time(),
-			Transactions: opaqueTxs,
+		BatchV2: derive.BatchV2{
+			Version:   2,
+			PayToAddr: common.Address{},
```

Comment:
So is this a way to reward the batcher? Is there some code to update that makes the transfer to this address?
Why is this field needed now? Because we have multiple batchers?

Reply:
Ah thanks.